src/app/header/header.component.html

```diff
@@ -79,7 +79,7 @@
               </a>
             </li>
             <li class="divider"></li>
-            <li>
+            <li id="createSpace">
```

Comment:
Good idea! I added a "header_" prefix to keep it consistent with other UI elements.